src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -31,7 +31,7 @@
 BUILD_NOT_SUPPORTED = 'Builds are only supported for managed registries.'
 
 
-def acr_build(cmd,
+def acr_build(cmd,  # pylint: disable=too-many-locals
```

Comment:
Instead of adding this pylint disable statement, you can extract some logic into a separate function.

Reply:
Could you link an example where a top level command has this many args and it's meaningful to extract it? I see 50 other places where people are doing this identical thing. FWIW, this isn't related to my change, I'm just trying to get CI passing since it'd be failing even without this.

Reply:
The existence of the pylint disable statement doesn't validate the necessity of it. In the process of the developing, there may be poor code style get merged in the code base, we should gradually rooting them out during the process of future developing.

The code style check exists for a reason: to stop gradually decaying of the code quality. Each rule is a summarize of the practice we collectively learned during the history of the software development. Unless there is a clear reason, I think we should follow it.

Here, I'm not asking you to extract the `arguments` but the `local variables`. The error indicates the function body is too big and uses too many local variables. The solution is to break the giant function into smaller ones. This is different from `too-many-argument` error.